phpBB/assets/javascript/core.js

```diff
@@ -1676,3 +1676,19 @@ $(function() {
 });
 
 })(jQuery); // Avoid conflicts with other libraries
+$(function(){
+
+jQuery( document ).ready(function() {
```

Comment:
Do not need all the jQuerys just use $ and wrap it in a self invoked function with jQuery passed into like the example above this in the same file

Reply:
Also why are you double wrapping with document ready?